docs/community.vmware.vmware_guest_module.rst

```diff
@@ -1420,10 +1420,8 @@ Parameters
                 <td>
                         <div>Flag the instance as a template.</div>
                         <div>This will mark the given virtual machine as template.</div>
-                        <div>Note, this may need to be done in a dedicated task invocation that is not making</div>
-                        <div>any other changes, e.g. you cannot change the state from powered-on to</div>
-                        <div>powered-off AND save as template in the same task. See vmware_guest.py source</div>
-                        <div>for more details.</div>
+                        <div>Note, this may need to be done in a dedicated task invocation that is not making any other changes. For example, user cannot change the state from powered-on to powered-off AND save as template in the same task.</div>
```

Comment:
For the record, this block is autogenerated from the content of the modules DOCUMENTATION block.

Reply:
@goneri Just for your information, @Akasurde already explained this [here](https://github.com/ansible-collections/community.vmware/pull/786#pullrequestreview-632882331).

@timblaktu Btw, you didn't have to close #786 and open a new PR. It would have been perfectly OK to add a new commit to your feature branch / PR. Actually, that's the way we generally work. It's easier for you because you don't have to open a new PR every time someone requests a change, and also it's easier for us to track if there is only one pull request.